modules/appeals_api/spec/support/shared_examples_contestable_issues.rb

```diff
@@ -2,183 +2,218 @@
+
+        it 'returns a 422' do
+          expect(response).to have_http_status(:unprocessable_entity)
+          expect(body['errors']).to be_an Array
+        end
       end
-    end
 
-    if options[:benefit_type].present?
-      context 'when benefit_type is unknown' do
+      context 'when X-VA-SSN and X-VA-File-Number are missing' do
+        let(:ssn) { nil }
+
         it 'returns a 422' do
-          tmp_options = options.dup
-          tmp_options[:benefit_type] = 'invalid benefit type'
-          get_issues(ssn: '872958715', options: tmp_options)
           expect(response).to have_http_status(:unprocessable_entity)
-          json = JSON.parse(response.body)
-          expect(json['errors']).to be_an Array
+          expect(body['errors']).to be_an Array
         end
       end
-    end
 
-    context 'when X-VA-SSN and X-VA-File-Number are missing' do
-      it 'returns a 422' do
-        get_issues(ssn: nil, file_number: nil, options: options)
-        expect(response).to have_http_status(:unprocessable_entity)
-        json = JSON.parse(response.body)
-        expect(json['errors']).to be_an Array
+      context 'when X-VA-SSN is invalid' do
+        let(:ssn) { '87295a71b' }
+
+        it 'returns a 422' do
+          expect(response).to have_http_status(:unprocessable_entity)
+          expect(body['errors']).to be_an Array
+        end
       end
-    end
 
-    context 'when X-VA-SSN has an invalid format' do
-      it 'returns a 422' do
-        get_issues(ssn: '87295a71b', options: options)
-        expect(response).to have_http_status(:unprocessable_entity)
-        json = JSON.parse(response.body)
-        expect(json['errors']).to be_an Array
+      context 'when X-VA-ICN is invalid' do
+        let(:icn) { '3939s31' }
+
+        it 'returns a 422' do
+          expect(response).to have_http_status(:unprocessable_entity)
+          expect(body['errors']).to be_an Array
+        end
       end
     end
 
-    context 'when X-VA-ICN has an invalid format' do
-      it 'returns a 422' do
-        get_issues(icn: '3939s31', options: options)
-        expect(response).to have_http_status(:unprocessable_entity)
-        json = JSON.parse(response.body)
-        expect(json['errors']).to be_an Array
+    context 'parameter validations' do
+      let(:cassette) { "caseflow/#{decision_review_type}/contestable_issues" }
+
+      before { get_contestable_issues }
+
+      unless base_path.include?('v1')
+        # Decision Reviews v1 doesn't use benefit type and its decision review type is constant
+        context 'when benefit_type is invalid' do
+          let(:ssn) { '872958715' }
+          let(:benefit_type) { 'invalid-benefit-type' }
+
+          it 'returns a 422' do
+            expect(response).to have_http_status(:unprocessable_entity)
+            expect(body['errors']).to be_an Array
+          end
+        end
+
+        context 'when both benefit type and decision review type are invalid' do
```

Comment:
This new test case is for the double-render bug - the others were all preexisting.